src/main/java/core/basesyntax/service/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.serviceimpl;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the empty line after the package declaration.